dracut/modules.d/99kiwi-lib/kiwi-partitions-lib.sh

```diff
@@ -404,6 +408,16 @@ function resize_wanted {
 #======================================
 # Methods considered private
 #--------------------------------------
+function _get_msdos_partition_start_sector {
+    # """
+    # read start sector from given partition
+    # """
+    local disk_device=$1
+    local partid=$2
+    sfdisk --dump "${disk_device}" |\
+        grep "${partid} :" | cut -f1 -d, | cut -f2 -d= | tr -d " "
```

Comment:
Just executed these commands in my host and I got something different than what I expected. This is what I got:

```
david@localhost:~> sudo sfdisk --dump /dev/sda 
label: gpt
label-id: CE4AA9A2-59DF-4DCC-B55A-A27A80676B33
device: /dev/sda
unit: sectors
first-lba: 34
last-lba: 500118158

/dev/sda1 : start=    17303552, size=   482814607, type=0FC63DAF-8483-4772-8E79-3D69D8477DE4, uuid=17B1F4B4-046E-4205-8592-A8284E5BC179
/dev/sda2 : start=      526336, size=    16777216, type=0657FD6D-A4AB-43C4-84E5-0933C84B4F4F, uuid=25D9EA9B-857B-470F-929D-6CD43842A71F
/dev/sda3 : start=        2048, size=      524288, type=C12A7328-F81F-11D2-BA4B-00A0C93EC93B, uuid=2C0AAEEA-455A-4CAD-8C2D-B034A4339769

david@localhost:~> sudo sfdisk --dump /dev/sda | grep 17B1 | cut -f2 -d= | tr -d " "
17303552,size
```
I would have expected `17303552` and not `17303552,size` as the output for this `_get_msdos_parition_start_sector` method. In fact I am surprised this is actually working. I am on Leap 15.2 (fresh install of today), sfdisk is from util-linux version 2.33.1.

Reply:
This is not the grep command as I have coded it. Try:

```
sudo sfdisk --dump /dev/sda | grep "1 :" | cut -f2 -d= | tr -d " "
```

partid matches the number of the partition and is handed over at another place in the initrd code.
As I said I have done several tests with good results

Reply:
Yes you are right I missed one `cut -f1 -d,` split on my test on the console, I did a wrong copy & paste. Yes I just verified it works as expected, on my host and on image. Sorry for the noise.